pyproject.toml

```diff
@@ -1,17 +1,30 @@
-[tool.poetry]
+[project]
+authors = [{ name = "Justin Mayer", email = "authors@getpelican.com" }]
+license = { text = "AGPLv3" }
+requires-python = ">=3.8,<4.0"
```

Comment:
In other projects I usually see a minimum of 3.8.1, so there must have been something not quite right with the initial 3.8 release, and we should probably follow their lead.